vaadin-upload-flow-parent/vaadin-upload-flow/src/main/java/com/vaadin/flow/component/upload/Upload.java

```diff
@@ -78,6 +78,55 @@ public UploadIcon() {
         }
     }
 
+    /**
+     * Defines the upload transmission format for files.
+     */
+    public enum UploadFormat {
+        /**
+         * Send files as raw binary data with Content-Type header set to the
+         * file's MIME type. This is the default format, which uses
+         * XMLHttpRequest (XHR) for file transmission.
```

Comment:
Both formats use XMLHttpRequest? I'm also not sure why we need to mention the headers, how does that matter for Flow developers?